airbyte-workers/Dockerfile

```
@@ -12,10 +12,10 @@ RUN apt-get update && apt-get install -y \
                           gnupg-agent \
                           software-properties-common
 RUN curl -fsSL https://download.docker.com/linux/debian/gpg | apt-key add -
-RUN add-apt-repository \
-       "deb [arch=${DOCKER_BUILD_ARCH}] https://download.docker.com/linux/debian \
-       $(lsb_release -cs) \
-       stable"
+RUN arch=$(arch | sed s/aarch64/arm64/ | sed s/x86_64/amd64/) && \
```

I haven't looked at the output from `arch`, but do you intend to replace `aarch` not `arch` here?

```suggestion
RUN arch=$(arch | sed s/arch64/arm64/ | sed s/x86_64/amd64/) && \
```

---

aarch64 are using arm64 in this case. Debian repo for aarch64 don't exists.